CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ CHANGELOG
 - Remove CloudFormation DescribeStacks API call from AWS Batch Docker entrypoint. This removes the possibility of job
   failures due to CloudFormation throttling.
 - Add support for io2 EBS volume type.
+- Add `iam_lambda_role` parameter under `cluster` section to enable the possibility to specify an existing role to 
```

Comment:
to specify an existing IAM role to be used with AWS Lambda functions that implement CustomResources in CloudFormation.